index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Hyunji Kim</title>
+  <link href ="styles/normalize.css" rel="stylesheet">
+  <link href ="styles/styles.css" rel="stylesheet">
+</head>
+<body>
+  <div class="background-image"></div>
+  <div class="content">
+    <header>
+      <span class="left">
+        <h2><a href="sites/start.html" target= "target-section" class="link">Hyunji Kim</a></h2>
+      </span><span class="right">
+        <nav>
+          <ul>
+            <li><a href="sites/about.html" target= "target-section">about</a></li>
+            <li><a href="sites/portfolio.html" target= "target-section">portfolio</a></li>
+            <li><a href="sites/hobby-blog.html" target= "target-section">hobby</a></li>
+          </ul>
+        </nav>
+      </span>
+    </header>
+
```

Comment:
Awesome usage of `<div>` and `<span>` here in the `<header>` section. You know the difference well!